### ISSUE TITLE
Problema de ejecución en NeuralForecast

### ISSUE THREAD

**Issue by AlineLPL** [Nixtla/neuralforecast#501]
Cuando trato de correr el código muestra que viene en el siguiente link https://nixtla.github.io/neuralforecast/examples/exogenous_variables.html#libraries
me arroja un error que no tenía antes al ejecutar la siguiente instrucción:
%%capture
nf = NeuralForecast(models=models, freq='H')
nf.fit(df=df,
       static_df=static_df)

Ya no entrena y arroja el siguiente error:

MisconfigurationException: The provided lr scheduler `StepLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.

¿Me pueden ayudar a entender que esta pasando?

**Comment by christianmacher** [comment]
Hello,
I also tried the examples in colab notebook 'Getting_Started' and got the same error. 
`MisconfigurationException: The provided lr scheduler `StepLR` doesn't follow PyTorch's LRScheduler API. You should override the `LightningModule.lr_scheduler_step` hook with your own logic if you are using a custom LR scheduler.`
It seems that something changed in PyTorch's API scheduler.

**Comment by FedericoGarza** [comment]
hey @AlineLPL and @christianmacher! Thanks for using neuralforecast. The problem probably arises due to the newest release of `torch` (2.0.0). Yesterday we updated the library to make it compatible with the latest version and `lightning >= 2.0`.

Could you try installing the new version from the main branch? Here’s the line: 

```
pip install "git+https://github.com/nixtla/neuralforecast.git@main"
```

We are preparing an official release to pip soon. :)

**Comment by felipevw** [comment]
Yes, installing the main branch in the noteboks solves the issue. Many thanks for the fix @FedericoGarza .

We can all go back at forecasting heteroskedastic time series!
